semcore/mini-chart/src/component/score/Line.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import createComponent, { Component, Root, sstyled, ComponentType } from '@semcore/core';
+import { Box, Flex, BoxProps } from '@semcore/flex-box';
+import resolveColorEnhance from '@semcore/utils/lib/enhances/resolveColorEnhance';
+
+import style from './line.shadow.css';
+
+export type CommonScoreProps = {
+  /**
+   * Value of score (in percents from 0 to 100) or count of selected segments (for Line with segments)
+   */
+  value: number;
+
+  /**
+   * Color of value
+   */
+  color?: string;
+
+  /**
+   * Flag to enable skeleton
+   * @default false
+   */
+  isLoading?: boolean;
```

Comment:
In other components we use just `loading`. Probably it's better to make it consistent

Reply:
changed